python/pylibcudf/pylibcudf/tests/test_hashing.py

```diff
@@ -115,6 +99,21 @@ def py_hasher(val):
     assert_column_eq(got, expect)
 
 
+def test_hash_column_xxhash32(pa_scalar_input_column, plc_scalar_input_tbl):
```

Comment:
@PointKernel This is failing for xxhash32 because there's a missing "hash combine" step. This doesn't occur for xxhash64 though. I think this is a bug in our xxhash32 implementation, it should match the `xxhash` package and not do a hash combine if we only have a single column. I'll look at this a little further.

Reply:
This isn't a bug but rather an improvement opportunity. The root cause is the unnecessary extra hash combine step in our device row hasher, which can be eliminated: https://github.com/rapidsai/cudf/blob/34e20451cf5452ecea74092dae3c6f5078ade0bd/cpp/include/cudf/table/experimental/row_operators.cuh#L1882-L1883

Reply:
Agreed. We discussed this on a call, and this will be an opportunity for follow-up work.